internal/config/function_export.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 Honey Science Corporation
```

Comment:
nit: we don't need to adjust the date on the existing ones, but we should update the date to 2020 on these headers